src/main/relation/read_csv_relation.cpp

```diff
@@ -36,16 +49,13 @@ ReadCSVRelation::ReadCSVRelation(const shared_ptr<ClientContext> &context, const
 	AddNamedParameter("columns", Value::STRUCT(std::move(column_names)));
 }
 
-ReadCSVRelation::ReadCSVRelation(const std::shared_ptr<ClientContext> &context, const string &csv_file,
+ReadCSVRelation::ReadCSVRelation(const std::shared_ptr<ClientContext> &context, const Value &input,
```

Comment:
Maybe have two separate constructors - one with a `string` and one with `vector<string>`? The single string can just call the `vector<string>` with a list.